### PR TITLE
lets see

### PR DESCRIPTION
**Pull request by 0xTranqui**

### CODE REVIEW

File: apps/site/config/relayConfig.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+export const relayWalletClient = createWalletClient({
+  account,
+  chain: river_j5bpjduqfv,
+  transport: http(process.env.RPC_URL),
+});
+
+const rawAccount = relayWalletClient.account;
+if (!rawAccount) {
+  // TODO: replace with viem AccountNotFoundError once its exported
```
Comment (salieflewis) [inline apps/site/config/relayConfig.ts:99]: I believe this error type is now being exported in the version of viem we're running.

File: apps/site/config/relayConfig.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  throw new Error("No account provided in nonce mamanger");
```
Comment (salieflewis) [inline apps/site/config/relayConfig.ts:100]: Spelling.

File: apps/site/config/relayConfig.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    account: address,
+    hasNonce,
+    nextNonce,
+    resetNonce,
+    shouldResetNonce,
+    mempoolQueue,
+  };
+}
+
+// Nonce mamanger setup
```
Comment (salieflewis) [inline apps/site/config/relayConfig.ts:87]: Spelling.

**Review by salieflewis — APPROVED** [review]
Looks good to me, left a few nits.

File: apps/site/config/relayConfig.ts (hunk 1 above)
Comment (0xTranqui) [reply apps/site/config/relayConfig.ts:99]: Screenshot 2024-01-28 at 4 17 02 PM
apparently not?

works if i do this but this is weird

import { AccountNotFoundError } from "viem/_types/errors/account";

**Pull request merged**